CheckboxItem/CheckboxItem.module.less

```diff
@@ -0,0 +1,32 @@
+// Checkbox.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/skin.less";
+
+.checkboxItem {
+	align-items: center;
+
+	.applySkins({
+		height: @agate-checkboxitem-height;
```

Comment:
@teckliew Curious if this is really necessary. When I added the label knobs to the sampler for this, the fixed height causes layout issues when the label is above or below. Does `CheckboxItem` need its own height value or should it rely on `Item`?

Reply:
We should leave it up to `Item`. This was leftover from before we use `Item`. Good catch.